plugins/module_utils/network/nxos/argspec/logging_global/logging_global.py

```diff
@@ -0,0 +1,278 @@
+                                "debugging",
+                            ],
+                        },
+                    },
+                },
+                "monitor": {
+                    "type": "dict",
+                    "options": {
+                        "state": {
+                            "type": "str",
+                            "choices": ["enabled", "disabled"],
+                        },
+                        "severity": {
+                            "type": "str",
+                            "choices": [
+                                "emergency",
+                                "alert",
+                                "critical",
+                                "error",
+                                "warning",
+                                "notification",
+                                "informational",
+                                "debugging",
+                            ],
+                        },
+                    },
+                },
+                "origin_id": {
+                    "type": "dict",
+                    "options": {
+                        "hostname": {"type": "bool"},
+                        "ip": {"type": "str"},
+                        "string": {"type": "str"},
+                    },
+                },
+                "rate_limit": {
+                    "type": "str",
+                    "choices": ["enabled", "disabled"],
+                },
+                "rfc_strict": {"type": "bool"},
+                "hosts": {
+                    "type": "list",
+                    "elements": "dict",
+                    "options": {
+                        "host": {"type": "str"},
+                        "severity": {
+                            "type": "str",
+                            "choices": [
+                                "emergency",
+                                "alert",
+                                "critical",
+                                "error",
+                                "warning",
+                                "notification",
+                                "informational",
+                                "debugging",
+                            ],
+                        },
+                        "facility": {"type": "str"},
+                        "port": {"type": "int"},
+                        "secure": {
+                            "type": "dict",
+                            "options": {
+                                "trustpoint": {
+                                    "type": "dict",
+                                    "options": {
+                                        "client_identity": {"type": "str"}
+                                    },
+                                }
+                            },
+                        },
+                        "use_vrf": {"type": "str"},
+                    },
+                },
+                "source_interface": {"type": "str"},
```

Comment:
is it str or list of str?

Reply:
It is a string since only one value is possible.

```
nxos-9kv-I5(config)# logging source-interface Ethernet 1/2
nxos-9kv-I5(config)# logging source-interface Ethernet 1/3
nxos-9kv-I5(config)# show running-config | in logging
logging source-interface Ethernet1/3
```

Reply:
okay